api/envoy/config/route/v3/route_components.proto

```diff
@@ -1564,3 +1567,74 @@ message QueryParameterMatcher {
     bool present_match = 6;
   }
 }
+
+// HTTP Internal Redirect :ref:`architecture overview <arch_overview_internal_redirects>`.
+// [#next-free-field: 6]
+message InternalRedirectPolicy {
+  // Configures :ref:`internal redirect <arch_overview_internal_redirects>` behavior.
+  enum InternalRedirectAction {
+    // Redirect responses are proxied to downstream without triggering Internal Redirect.
+    // This is the default value.
+    PASS_THROUGH_INTERNAL_REDIRECT = 0;
```

Comment:
Is this needed anymore? If there is no policy (the default) there would be pass through?

Reply:
Removed. Rely on the presence of internal_redirect_policy to inform the enablement.